### ISSUE TITLE
allow modification and report

### ISSUE THREAD

**Issue by ManjotS** [KhalisFoundation/sttm-desktop#204]
Please select which platform you are using:-
Desktop Version

Feedback Type
Enhancements

What went wrong?
In the old Sikhi to the Max, the raw generated powerpoint was available. This is no longer available, so we dont have the opportunity to modify mistranslations or mistransliterations. With the old program, we could quickly update the slide if there was an error, now we cant do that.

What should have happened instead?

How can the issue be recreated by our team?
Slides are hard locked. There are still plenty of errors, that we could have fixed on the fly, but no longer can.

Which web browser are using?

Operating System

Name
Herman Basra

**Comment by ManjotS** [comment]
We should address this as "Suggest a change".

It should basically turn the preview window into editable fields (or in single screen, the slide itself) with enter saving it.

That change should submit to our backend control panel as a suggested user change (to be implemented), and temporarily update the slide for the current session.

We can also add a setting to turn off having live changes submitted. Changes should queue in a local db if there is no internet connection with a max of 50.

**Comment by maneetpaul** [comment]
I don't think we should allow live editing for current sessions. But I do agree there should be a "Report an Error" button somewhere

**Comment by prebhdevsingh** [comment]
Will keep this in the backlog but lower priority since the earlier STTM2 feature was mostly used to edit typos in Gurbani and wrong translations. We've done a lot of improvements with banidb. We will need to see if users really need this option.

I'd love to provide a ability where users can submit a change to banidb directly for approval.
